server/api/router.go

```diff
@@ -358,6 +352,14 @@ func createRouter(prefix string, svr *server.Server) *mux.Router {
 	registerFunc(clusterRouter, "/admin/unsafe/remove-failed-stores/show",
 		unsafeOperationHandler.GetFailedStoresRemovalStatus, setMethods(http.MethodGet), setAuditBackend(prometheus))
 
+	if !svr.IsAPIMode() {
```

Comment:
What if "pd-server services api,tso XXXX"

Reply:
nice catch. I think the current implementation cannot well support this kind of input, we need to use a flag to specify the mode instead of using subcommand.

Reply:
use alias to solve this problem